internal/models/stepdef.go

```diff
@@ -36,10 +36,10 @@ type StepDefinition struct {
 var typeOfContext = reflect.TypeOf((*context.Context)(nil)).Elem()
 
 // Run a step with the matched arguments using reflect
-func (sd *StepDefinition) Run(ctx context.Context) (context.Context, interface{}) {
+func (stepDef *StepDefinition) Run(ctx context.Context) (context.Context, interface{}) {
```

Comment:
What's a `def`?

Reply:
Given my other comment below on method receivers, I'd personally prefer this be kept as at least `sd` if not fully changed to `s`.

Reply:
Also to piggyback on @mpkorstanje 's comment, if I'm assuming their intent correctly, the file could probably use being renamed to `stepdefinition.go`

Reply:
Good points both - done!